src/content/ui/animations/index.md

```diff
@@ -186,10 +164,247 @@ or might partially or completely overlap.
   <img src="/assets/images/docs/ic_new_releases_black_24px.svg" alt="this doc is new!"> NEW<br>
 {% endcomment -%}
```

Comment:
So, I see other places where we now use "-%" to end the comment. Is that a new thing? Anyway, this endcomment doesn't seem to have a beginning comment?